src/components/MessagesList/MessagesList.vue

```diff
@@ -1238,6 +1245,15 @@ export default {
 			// scroll down by the height difference
 			this.$refs.scroller.scrollTop += heightDiff
 		},
+
+		updateTasksCount() {
+			if (!this.$refs.scroller) {
+				return
+			}
+			const tasksDoneCount = this.$refs.scroller.querySelectorAll('.checkbox-content__icon--checked')?.length
```

Comment:
Another way could be to check message store for messages (`- [ ] ` and `- [x] `?

In case there would be an opportunity to render not all messages (virtual scroller)

Reply:
Performance-wise, it is more expensive as you will loop through messages.
Accuracy-wise, not all `- [ ]` are considered a checklist, maybe we need to do the same check when clicking on a list item to change the state.